lib/X25519KeyPair.js

```diff
@@ -0,0 +1,321 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const forge = require('node-forge');
+const {util: {binary: {base58}}} = forge;
+const {LDKeyPair} = require('crypto-ld');
+const {base58Decode} = require('crypto-ld/lib/util');
+const ed2curve = require('ed2curve');
+const nacl = require('tweetnacl');
+
+class X25519KeyPair extends LDKeyPair {
+  /**
+   * An implementation of x25519
+   * [X25519 Key Agreement 2019]{@link https://w3c-dvcg.github.io/} representation
```

Comment:
Is that the best link?  Not clear what is x25519 related there.

Reply:
I suspect that link was put in there in an aspirational capacity - as a reminder to create an X25519 sig suite repo, in that org?